resources/views/partials/content-page-settings.blade.php

```diff
@@ -1,11 +1,20 @@
-<ul class="link-list">
-  @foreach(get_posts([
-    'post_type' => 'page',
-    'posts_per_page' => -1,
-    'orderby' => 'menu_order',
-    'order' => 'ASC',
-    'post_parent' => $post->ID
-    ]) as $child)
-  <li class="link-list__item"><a href="{{ get_permalink($child->ID) }}">{{ $child->post_title }}</a></li>
-  @endforeach
-</ul>
+<form method="post">
+  <p><strong>{{ __('Allow the Resource Library to track what resources you’ve viewed.', 'coop-library') }}</strong></p>
+  <span class="disclosure-label" hidden>{{ __('How this changes what I see', 'coop-library') }}</span>
+  <div class="details">
+      <p>{{ __('If you allow the resource library to know this, you would be able to see resources that are recently viewed by you, related to your recently viewed, and contribute your information to create the most viewed resource section seen by the larger community.', 'coop-library') }}</p>
```

Comment:
Suggestion to tweak to copy. I'm afraid it is getting longer, but it seems like it should be addressed whenever user tracking is involved:

"By choosing 'Allow' the Resource Library to track the resources you view, you will:
* see a list of your recently viewed resources on the home page,
* see related resources, and
* contribute to the most viewed articles seen on the home page.

The Resource Library will only use this information for these features, and does not collect any personally identifiable information.

By choosing 'Don't allow', you will be able to continue to use the Resource Library fully, with the exception of the features mentioned previously."

@cherylhjli @dayotte have any input?

Reply:
this sounds good Jon!